apps/okreads/browser/src/app/app.component.html

```diff
@@ -3,8 +3,9 @@
     <header class="header">
       <div class="header--content">
         <h1>okreads</h1>
-        <div>
+        <div tabindex="0" role="button">
```

Comment:
No need to provide role="button" here.
Button tag will handle it automatically, if you run accessibility tool then it will read it as button only.